Clarify help text on scaling dashboard (#1666)
grafana/mimir: docs/sources/operators-guide/visualizing-metrics/dashboards/scaling/index.md

> # Grafana Mimir Scaling dashboard

Reviewer: ```suggestion
# Grafana Mimir scaling dashboard
```

> The Scaling dashboard suggests services that should be scaled up in the event of a failure for one of several specific reasons.

Reviewer: ```suggestion
The scaling dashboard displays services that you can scale up, in the event of a failure for one or more specific reasons.
```
I like this comma here because it separates two preposition "up in".
